build_projects/dotnet-host-build/sign.proj

```diff
@@ -47,6 +47,18 @@
       <FilesToSign Include="$(IntermediateOutputPath)sharedFrameworkPublish/shared/**/Microsoft.*.dll">
         <Authenticode>$(CertificateId)</Authenticode>
       </FilesToSign>
+      <FilesToSign Include="$(IntermediateOutputPath)sharedFrameworkPublish/shared/**/mscorlib.dll">
```

Comment:
@chcosta this is one place to update once you switch to buildtools. We should do the signing similar to corefx but we need to do it after we run crossgen on the binaries but before packaging.